doc/effects.md

```diff
@@ -46,6 +46,24 @@ square.addEffect(ScaleEffect(
 ));
 ```
 
+## RotationalEffect
```

Comment:
The other effects are using the verb instead of the noun, I think we should name it RotateEffect, to keep on the same pattern, what do you think?

Reply:
Fixed